pos_laundry_management/static/src/css/pos.css

```diff
@@ -70,3 +70,21 @@
     margin-right: 5px;
     color: #555555;
 }
+
+.pos .popup.popup-receipt .button.state_cancel,
+.pos .popup.popup-receipt .button.state_cancel:active{
+    float:right;
+    width: 110px;
+    height: 40px;
+    line-height:40px;
+    text-align:center;
+    border-radius: 2px;
+    margin-top:10px;
+    margin-right:10px;
+    font-size:   14px;
+    font-weight: bold;
+    cursor: pointer;
```

Comment:
@KolushovAlexandr I think it's better to use `cursor: unset;` because the button is inactive